### PR TITLE
Update SSW

### PR DESCRIPTION
**Pull request by kahlouie**
## Description

This uses a new SSW template, adds mileage and totals for SIT and non-SIT expenses. It also separates the signature from the date for the service member.

## Reviewer Notes

Please make sure that all the values are in the right spot. There's a lot of pieces that got moved.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
go run cmd/generate_shipment_summary/main.go  -move <uuid> -debug
```
I would use a move that you've created fully from scratch.

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* https://www.pivotaltracker.com/story/show/167907725

## Screenshots

[shipment_summary_worksheet_stripped.pdf](https://github.com/transcom/mymove/files/3522147/shipment_summary_worksheet_stripped.pdf)

### CODE REVIEW

File: pkg/models/shipment_summary_worksheet.go [hunk 1]
```diff
@@ -567,6 +592,7 @@ func FormatSitExpenses(movingExpenseDocuments MovingExpenseDocuments) (Formatted
 	return FormattedSitExpenses{
 		TotalMemberPaidSIT: storageExpenses.StorageMemberPaid,
 		TotalGTCCPaidSIT:   storageExpenses.StorageGTCCPaid,
+		TotalPaidSIT:       FormatDollars(subTotals["TotalPaid"]),
```
Comment (jim) [inline pkg/models/shipment_summary_worksheet.go]: `FormatSitExpenses` is pretty hard to follow. Not sure you want to pull refactoring it into this PR, but if `SubTotalExpenses` returned a struct instead of a `map[string]` we could avoid some of the hoops this code is jumping through to deal with that data structure.

File: pkg/rateengine/rateengine.go [hunk 2]
```diff
@@ -36,6 +36,7 @@ type CostComputation struct {
 	SITDiscount unit.DiscountRate
 	Weight      unit.Pound
 	ShipmentID  uuid.UUID
+	Miles       int
```
Comment (jim) [inline pkg/rateengine/rateengine.go]: Can we create a `unit.Miles` type here to represent this distance? All of the other values in this struct are specific types that indicate the unit of their measure.

File: pkg/models/shipment_summary_worksheet.go [hunk 3]
```diff
@@ -20,13 +20,14 @@ import (
 )
 
 // FormatValuesShipmentSummaryWorksheet returns the formatted pages for the Shipment Summary Worksheet
-func FormatValuesShipmentSummaryWorksheet(shipmentSummaryFormData ShipmentSummaryFormData) (ShipmentSummaryWorksheetPage1Values, ShipmentSummaryWorksheetPage2Values, error) {
+func FormatValuesShipmentSummaryWorksheet(shipmentSummaryFormData ShipmentSummaryFormData) (ShipmentSummaryWorksheetPage1Values, ShipmentSummaryWorksheetPage2Values, ShipmentSummaryWorksheetPage3Values, error) {
 	page1 := FormatValuesShipmentSummaryWorksheetFormPage1(shipmentSummaryFormData)
 	page2, err := FormatValuesShipmentSummaryWorksheetFormPage2(shipmentSummaryFormData)
+	page3 := FormatValuesShipmentSummaryWorksheetFormPage3(shipmentSummaryFormData)
```
[the remark this answers is not in the document]
Comment (kahlouie) [reply pkg/models/shipment_summary_worksheet.go]: Page 2 is the only one that calls the rate engine. The rest should have static data.

File: pkg/models/shipment_summary_worksheet.go (hunk 1 above)
Comment (kahlouie) [reply pkg/models/shipment_summary_worksheet.go]: That's a great idea. Can we leave this as is and I follow on with a refactor pr after? It looks like our backlog is pretty empty for the rest of this sprint anyway.

File: pkg/rateengine/rateengine.go (hunk 2 above)
Comment (LeDeep) [reply pkg/rateengine/rateengine.go]: You don't need to add `Miles` to the struct here. We have access to `Mileage` from the `LinehaulCostComputation` struct. In the `ShipmentSummary` where you're calling `actualCost.Miles`, you can instead make a call to `actualCost.Mileage`.

If you still wanted to convert the type to `unit.Miles`, you can do that in `Linehaul.go`.
